part-2-server/server.js

```diff
@@ -0,0 +1,44 @@
+var http = require('http');
+var fs = require('fs');
+
+
+var server = http.createServer(function(request, response){
+	if(request.url === "/"){
+		request.url = request.url+"index.html";
+	}//////////////////////////////////////////////////////////////////
+	/////////// this is not needed but i added it due to the way the question
+	/////////// was written as long as all of your a tags are writen with the
+	/////////// same name as your file////////////////////////////////
+	//////////////////////////////////////////////////////////////////
+	else if(request.url === '/page-2'){
+		request.url = request.url+".html";
+	}else if(request.url === '/page-3'){
```

Comment:
If url is '/page-3' it should still response with 'page-2.html', according to the instructions